src/components/VtkThreeView.vue

```diff
@@ -326,27 +326,33 @@ export default defineComponent({
           light.setPositional(false);
         }
 
-        const dims = image.getDimensions();
-        const spacing = image.getSpacing();
-        const spatialDiagonal = vec3.length(
-          vec3.fromValues(
-            dims[0] * spacing[0],
-            dims[1] * spacing[1],
-            dims[2] * spacing[2]
-          )
-        );
+        if (animating) {
+          mapper.setSampleDistance(0.75);
+          mapper.setMaximumSamplesPerRay(1000);
+          mapper.setGlobalIlluminationReach(0);
+        } else {
+          const dims = image.getDimensions();
+          const spacing = image.getSpacing();
+          const spatialDiagonal = vec3.length(
+            vec3.fromValues(
+              dims[0] * spacing[0],
+              dims[1] * spacing[1],
+              dims[2] * spacing[2]
+            )
+          );
 
-        // Ensure that the maximum samples per ray are not exceeded.
-        let sampleDistance = mapper.getSampleDistance();
-        if (
-          spatialDiagonal / sampleDistance >
-          mapper.getMaximumSamplesPerRay()
-        ) {
-          sampleDistance =
-            spatialDiagonal / (mapper.getMaximumSamplesPerRay() - 1);
+          // Use the average spacing for sampling by default
+          let sampleDistance = spacing.reduce((a, b) => a + b) / 3.0;
```

Comment:
```suggestion
          let sampleDistance = spacing.reduce((a, b) => a + b) / spacing.length;
```
Could be 2D image?

Reply:
vtkImageData spacing is always a 3-tuple even if it is 2D.

Reply:
🧑‍🎓

Reply:
Can we reduce this even further?   Is there a nyquest-ish thing - perhaps / 4 is a reasonable limit?

Reply:
It takes the average dataset spacing as the sample distance. This ensures that smaller differences in intensities are captured as well.

Reply:
The images still looking overly "grainy" to me, and rendering is nearly instantaneous for both Performance and Quality settings.   Looking for ways to improve visualization even if it costs more time.

So, performance is really quite poor.   We have level-of-detail that keeps things interactive.   Even when on performance rendering, can we reduce the grainy-ness (black dots/noise on bone surface)?
<img width="1268" alt="image" src="https://user-images.githubusercontent.com/279727/223469642-142c2254-c474-4898-961b-89975b309288.png">
and

Quality setting produces a better image, but there are still banding/aliasing artifacts and some grainy-ness.  The LAO sliders (kernel size and radius) should be able to improve the rendering, but they seem to have no effect.  Overall, if we can have more computation produce a better image, we should.    If this is the best image LAO can produce with this data, then ok, but I hope vtk.js can do better...
<img width="1268" alt="image" src="https://user-images.githubusercontent.com/279727/223469910-6f439e34-eb38-4d3f-8c12-3365ed9f9280.png">